### DISCUSSION TITLE
(Recommended) Docker installation

### DISCUSSION THREAD

**Discussion by jhermann** [devpi/devpi#999] [General]
I found quite a few "install via docker" repositories on GitHub, is there one the community recommends?

Are there plans for an "official" one, or would you accept one into the official repos?

**Comment by fschulze** [comment]
I don't use docker. I use FreeBSD jails and am looking into NixOS for Linux based stuff. I can only point to this: https://github.com/devpi/devpi/issues/439#issuecomment-1168518972

**Comment by jhermann** [comment]
Thanks for the pointer, I'll take a look at https://github.com/JonasAlfredsson/docker-devpi
